### PR TITLE
add hard_bit method to DecodeFrom trait

### PR DESCRIPTION
**Pull request by scd31**
As previously discussed. Feel free to release a new version after this, whenever you get a chance!

### CODE REVIEW

File: src/decoder.rs [hunk 1]
```diff
@@ -35,6 +35,10 @@ pub trait DecodeFrom:
     fn saturating_add(&self, other: Self) -> Self;
     /// Saturating sub
     fn saturating_sub(&self, other: Self) -> Self;
+    /// Get hard-bit equivalent
+    fn hard_bit(&self) -> bool {
```
Comment (adamgreig) [inline src/decoder.rs]:
```suggestion
    /// Hard-information decoding for T, where negative values are a 1 bit and return `true`.
    #[inline]
    fn hard_bit(&self) -> bool {
```

**Review by adamgreig — APPROVED** [review]

**Comment by scd31** [comment]
@adamgreig Mind cutting a new release whenever you have a sec?

**Comment by adamgreig** [comment]
Sorry for the wait, I started investigating a few performance tweaks in the min-sum decoder but ran out of time before I got any conclusive answers. I've published the panic fixes, the new hard_bit() method, and some trivial updates to use it in `decode_ms` as v1.2.0. Thanks for your contributions!

**Comment by scd31** [comment]
No worries! Didn't want to rush you, just thought you might have forgotten

**Pull request merged**